scripts/integration-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source /scripts/common.sh
+source /scripts/bootstrap-helm.sh
+
+
+run_tests() {
+    echo Running tests...
+
+    wait_pod_ready polkadot-watcher-registrar
+}
+
+teardown() {
+    helm delete polkadot-watcher-registrar
+}
+
+main(){
+    if [ -z "$KEEP_W3F_POLKADOT_WATCHER" ]; then
+        trap teardown EXIT
+    fi
+
+    /scripts/build-helm.sh \
```

Comment:
What we have been doing lately is moving all the value setting to helmfile.d, and keep here just `/scripts/build-helm.sh` like here for instance https://github.com/w3f/purchase-web-app-deployment/blob/master/scripts/integration-tests.sh#L31 let me know wdyt.